assets/src/graph/policies.graphql

```diff
@@ -0,0 +1,66 @@
+fragment PoliciyConstraint on PolicyConstraint {
+  cluster {
+    ...Cluster
+  }
+  description
+  id
+  insertedAt
+  name
+  recommendation
+  ref {
+    kind
+    name
+  }
+  updatedAt
+  violationCount
+  violations {
```

Comment:
we probably don't want to sideload this on every query of a `PolicyConstraint`.  Ideally we'd only use them on the detail view of a constraint unless it's absolutely necessary.